backend/src/StamAcasa.Common/DTO/UserInfo.cs

```diff
@@ -7,6 +7,7 @@ public class UserInfo {
         [JsonIgnore]
         public int? ParentId { get; set; }
         public string Name { get; set; }
+        public string Email { get; set; }
```

Comment:
This information should not be displayed in API, we should add `JsonIgnore` to this field.

Reply:
Thanks, nice catch! Sorry about it, I pushed a fix.